FWCore/ParameterSet/interface/ParameterWildcardWithSpecifics.h

```diff
@@ -0,0 +1,42 @@
+
+#ifndef FWCore_ParameterSet_ParameterWildcardWithSpecifics_h
+#define FWCore_ParameterSet_ParameterWildcardWithSpecifics_h
+
+#include <string>
+#include <map>
+#include "FWCore/ParameterSet/interface/ParameterWildcardBase.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+namespace edm {
+
+  class ParameterSet;
+  class DocFormatHelper;
+
+  class ParameterWildcardWithSpecifics : public ParameterWildcardBase {
```

Comment:
Could you document the usage of this class in
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Wildcards_More_Details
after this PR has been merged?

Reply:
OK